device/dell/x86_64-dell_s6000_s1220-r0/pmon_daemon_control.json

```diff
@@ -1,3 +1,4 @@
 {
-    "skip_ledd": true
+    "skip_ledd": true,
+    "skip_syseepromd": true
```

Comment:
We eventually want all platforms to implement platfrom API 2,0 support, which will allow syseepromd to function properly. If we silence these messages here, We need to make sure Dell is aware of this temporary change, so that it can be reverted once the support is present.